internal/metrics/mock.go

```diff
@@ -0,0 +1,36 @@
+package metrics
+
+import (
+	"fmt"
+	storage "github.com/mrkovshik/yametrics/internal/storage/agent"
+	"log"
+	"math/rand"
+	"time"
+)
+
+type MockMetrics struct {
```

Comment:
SUGGESTION:
https://github.com/uber-go/mock

Reply:
I read gomock docs, and honestly, I don't see how is it going to work here, maybe i'm gonna need to discuss it on 1:1 meeting. Anyway, as long as this comment marked as "SUGGESTION", I guess I can leave it as is by now